sdks/python/apache_beam/runners/worker/opcounters.pxd

```diff
@@ -21,6 +21,22 @@ cimport libc.stdint
 from apache_beam.utils.counters cimport Counter
 
 
+cdef class TransformIOCounter(object):
+  cpdef add_bytes_read(self, libc.stdint.int64_t n)
```

Comment:
<!--new_thread; commit:9c3a73fcac33b56a026e4ee575722c94c34cf64f; resolved:0-->
The `check_step` method isn't declared here; can you add it?